user/hasher.go

```diff
@@ -12,17 +15,23 @@ func generateUniqueHash(strings []string, length int) (string, error) {
 
 	if len(strings) > 0 && length > 0 {
```

Comment:
We should fail if length > 64 (sha256 always returns 256 bits, which is 64 hexdecimal characters).  This is just being paranoid.